tj2-reco.py

```diff
@@ -9,6 +9,12 @@
 
 python3 tj2-reco.py   --runno $run  --gearfile $gearfile 
 
+
+To activate the clusterderDB for position reconstruction instead of center of gravity, 
+run command with prefix "_clustdb"
+
+python3 tj2-reco.py   --runno $run  --gearfile $gearfile --prefix _clustdb
```

Comment:
use this prescription to activate the clusterDB based position reconstruction. 

the time for calibration will be longer (x2 or similar) but the estimation of the tracking extrapolation errors are better. use this as baseline whenever quoting intrinsic position resolutions (angular scans)